### ISSUE TITLE
Extension doesn't load; can't find module 'is-color'

### ISSUE THREAD

**Issue by chrisant996** [j0meinaster/bracket-peek#10]
The 1.2.0 release doesn't seem to load.  When launching `code` normally, there's no warning but the extension's functionality doesn't activate.  When launching any extension into the debugger, the following error message pops up:
```
Activating extension 'jomeinaster.bracket-peek' failed: Cannot find module 'is-color'
Require stack:
- c:\Users\chrisant\.vscode\extensions\jomeinaster.bracket-peek-1.2.0\src\config\index.js
- c:\Users\chrisant\.vscode\extensions\jomeinaster.bracket-peek-1.2.0\src\extension.js
- c:\Users\chrisant\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Users\chrisant\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Users\chrisant\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork.js.
```

**Comment by j0meinaster** [comment]
Run:

npm install 

:)

**Comment by chrisant996** [comment]
I have done, multiple times. The extension still fails to load.

But I'm just installing the extension from the marketplace (not building it), so I shouldn't have to run npm install at all, right?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by j0meinaster** [comment]
Ah damn it,

i was not aware that node_modules were ignored. Thanks for figuring out.

Fixed in 1.2.1
